common/models/user.js

```diff
@@ -545,25 +545,30 @@ module.exports = function(User) {
    * ```js
    * {
    *   type: 'email',
-   *   mailer: anEmailModel,
+   *   mailer: function(verifyOptions, options, cb) {cb(null, someMailerFunction); }
    *   to: 'test@email.com',
    *   from: 'noreply@email.com'
    *   subject: 'verification email subject',
-   *   text: 'verification email text',
+   *   text: 'Please verify your email by opening this link in a web browser',
    *   headers: {'Mime-Version': '1.0'},
    *   template: 'path/to/template.ejs',
+   *   templateFn: function(verifyOptions, options, cb) { cb(null, "some body template"); }
    *   redirect: '/',
-   *   verifyHref: '',
+   *   verifyHref: 'http://localhost:3000/api/user/confirm',
    *   host: 'localhost'
    *   protocol: 'http'
    *   port: 3000,
    *   restApiRoot= '/api',
-   *   tokenGenerator: function (user, cb) { cb("random-token"); }
+   *   generateVerificationToken: function (user, cb) { cb(null, "random-token"); }
    * }
    * ```
    *
-   * The default options can be modified in your custom user model json definition
-   * using `settings.verifyOptions` or programmatically like follows:
+   * NOTE: param `to` internally defaults to user's email but can be overriden for
+   * test purposes or advanced customization.
+   *
+   * Static default params can be modified in your custom user model json definition
+   * using `settings.verifyOptions`. Any default param can be programmatically
+   * like follows:
```

Comment:
> Any default param can be programmatically like follows

I think you are missing a verb between "programmatically" and "like". I mean what can be programmatically done?